src/wp-includes/default-filters.php

```diff
@@ -135,6 +135,7 @@
 add_filter( 'the_content', 'wpautop'            );
 add_filter( 'the_content', 'shortcode_unautop'  );
 add_filter( 'the_content', 'prepend_attachment' );
+add_filter( 'the_content', 'wp_make_content_images_responsive' );
```

Comment:
I really like the other changes to function names you did!  I think this one is probably too long, but naming shouldn't stop us from an initial patch.